MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -174,7 +166,7 @@ public class NavigationDirections: Directions {
      
      - parameter options: A `RouteOptions` object specifying the requirements for the resulting routes.
      - parameter offline: Determines whether to calculate the route offline or online.
-     - parameter completionHandler: The closure (block) to call with the resulting routes. This closure is executed on the application’s main thread.
+     - parameter completionHandler: The closure (block) to call with the resulting routes. This closure is executed on the application’s main thread. If called `NavigationDirections` instance is deallocated before route calculation is finished - completion won't be called.
```

Comment:
Not familiar with context of this problem, so just minor comment from me: probably better to limit length of lines to around 100-120, just for better readability.